operations/configuration.md

```diff
@@ -406,17 +406,17 @@ Returns all tax environments supported by the API.
 
 | Property | Type |  | Description |
 | --- | --- | --- | --- |
-| `Code` | string | required | Code of the taxation. |
-| `TaxEnvironmentCode` | string | required | Code of the tax environment. |
+| `Code` | string | required | Code of the taxation. Corresponds to the `TaxationCode` seen in the [Tax rate](#tax-rate) object. |
+| `TaxEnvironmentCode` | string | required | Code of the tax environment. Corresponds to the `Code` seen in the [Tax environment](#tax-environment) object and in [Get configuration](#get-configuration)|
```

Comment:
Usually we comment foreign keys like `Code of the [Tax environment](#tax-environment) the taxation belongs to.` Or something along this lines. check any other foreign key property.